src/graphs.jl

```diff
@@ -269,39 +327,28 @@ const graph_aliases = Dict(:curvature_scalar => [:curvaturescalar,:curvature],
             g.args = (LightGraphs.Graph(g.args[1]),)
         end
     end
-    # Process GraphRecipes aliases, keywords that are already in Plots.jl have to be dealt
-    # with individually, since they cannot be deleted from the plotattributes
-    # dictionary.
-    if haskey(plotattributes, :markercolor)
-        plotattributes[:nodecolor] = nodecolor
-        nodecolor = plotattributes[:markercolor]
-        delete!(plotattributes, :nodecolor)
-    end
-    if haskey(plotattributes, :markersize)
-        plotattributes[:nodesize] = nodesize
-        nodesize = plotattributes[:markersize]
-        delete!(plotattributes, :nodesize)
+
+    # To process aliases that are unique to graphplot, find aliases that are in
+    # plotattributes and replace the attributes with their aliases. Then delete the alias
+    # names from the plotattributes dictionary.
+    @process_aliases plotattributes graph_aliases
+    for arg in keys(graph_aliases)
+        remove_aliases!(arg, plotattributes, graph_aliases)
     end
-    if haskey(plotattributes, :markershape)
-        plotattributes[:nodeshape] = nodeshape
-        nodeshape = plotattributes[:markershape]
-        delete!(plotattributes, :nodeshape)
+    # The above process will remove all marker properties from the plotattributes
```

Comment:
I'm not sure I really understand the motivation here. I get that you want people to be able to use the `node`-arguments rather than `marker`, but why not keep the internal consistency wiht Plots?

Reply:
Good question. Essentially I have to delete all aliases from the `plotattributes` dictionary, otherwise the recipe will complain that there are unrecognized attributes. In the for loop, `marker*` are all just regular GraphRecipes aliases. Although really they are not just aliases to `node*`, they are actual plot attributes, like you point out. That is why I manually repopulate `plotattributes` with the `marker*` properties in the lines just below 338.

Reply:
and wouldn't it be easier to do it the other way around, having `node` be alias for `marker` not the other way around?

Reply:
Yes, it might be slightly easier that way. Although, from a user perspective, both ways are identical. I just want to emphasize that nodes are different from markers even though they share a lot (all of the same) of properties. I will admit that your thinking is slightly more logical, I may have slightly overplayed the philosophical differences in my head. In any case, making node an alias of marker is a non breaking change, so it can be done at any time.

Reply:
great